coalib/results/SourceRange.py

```diff
@@ -126,6 +126,40 @@ def __json__(self, use_relpath=False):
             _dict['file'] = relpath(_dict['file'])
         return _dict
 
+    def __str__(self):
+        """
+        Creates a string representation of the SourceRange object
```

Comment:
\+ you could enclose SourceRange into double backticks: `´´SourceRange´´` (I used `´` here because github doesn't allow to escape `), so it gets rendered as code